How-To-Setup-Internal-Package-Repository.md

```diff
@@ -211,17 +193,18 @@ Jenkins requires several PowerShell scripts to automate the processes. Create a
       [Parameter(Mandatory)]
       [string]
       $RemoteRepo
+
   )
 
   . .\ConvertTo-ChocoObject.ps1
 
   Write-Verbose "Getting list of local packages from '$LocalRepo'."
-  $localPkgs = choco list --source $LocalRepo | Select-Object -Skip 1 | Select-Object -SkipLast 1 | ConvertTo-ChocoObject
+  $localPkgs = choco list --source $LocalRepo -r | ConvertTo-ChocoObject

```

Comment:
Can you use the full parameter name here as per the [style guidelines](https://chocolatey.org/docs/commands-reference#scripting-integration-best-practices-style-guide)?

Reply:
Addressed